src/fmod_studio.rs

```diff
@@ -66,4 +67,12 @@ impl FmodStudio {
 
         studio
     }
+
+    pub fn build_audio_source_from_path(&self, path: &str) -> BevyFmodAudioSource {
```

Comment:
This name could be better. I can't think of anything better to save my life.

Reply:
Since `path` is already in the arguments I think it's justified to call it `build_audio_source`